src/calendar/calendar-day.style.tsx

```diff
@@ -0,0 +1,174 @@
+import { Color } from "../color";
+import { TextStyleHelper } from "../text";
+import { GenerateDayClass, VariantDay } from "./types";
+import styled, { css } from "styled-components";
+
+interface DayNumberProps {
+    $variant: VariantDay;
+    $disabled: boolean;
+}
+
+interface GrowDayCellProps extends GenerateDayClass {}
+
+interface CircleProps extends Pick<DayNumberProps, "$variant"> {
+    $position: "left" | "right";
+}
+
+interface InteractiveCircleProps extends Omit<DayNumberProps, "$variant"> {}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const WeekDaysContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    margin-bottom: 0.5rem;
+`;
+
+export const WeekCell = styled.div`
+    ${TextStyleHelper.getTextStyle("H6", "semibold")}
+    height: 1.625rem;
+    width: 4.875rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: ${Color.Neutral[1]};
+    pointer-events: none;
+    user-select: none;
+`;
+
+export const CalendarDaySection = styled.div`
+    display: flex;
+    justify-content: space-around;
+    margin-bottom: 0.25rem;
+`;
+
+export const GrowDayCell = styled.div<GrowDayCellProps>`
+    display: flex;
+    position: relative;
+    height: 2.5rem;
+    align-items: center;
+    justify-content: center;
+
+    ${(props) => {
+        const { $selected } = props;
+
+        if ($selected) {
+            return css`
+                ${LeftCell} ${Circle} {
```

Comment:
Arh.. I tried to create LeftCircle but some error throw from styled component.

```
const LeftCircle = LeftCell > LeftCircle;
```

Reply:
oh, I meant creating new components. like `const LeftCircle = styled(Circle)`

Reply:
Arh. how we indicated that is Left?

Because the Circle will shared in 
```
<LeftCell>
  <LeftCircle />
</LeftCell />

<RightCell>
  <RightCircle />
</RightCell />
```

Reply:
is this your expected?

```
 ${LeftCell} ${LeftCircle} {
    background-color: ${Color.Accent.Light[5]};
    border: 1px solid ${Color.Primary};
}
```

Reply:
yep pretty much, but if you just want to style `LeftCircle`, then no need to specify `LeftCell`

Reply:
Let me know if i'm wrong.

I need to select which Circle i selected.

In Dom, there has two Circle, 
1. LeftCell Circle
2. RightCell Circle

If i just (below code), it selected both <Circle /> element.
```
 ${LeftCircle} {
    background-color: ${Color.Accent.Light[5]};
    border: 1px solid ${Color.Primary};
}
```

Reply:
Opps. Yes.

You are right and explanation. thank for your time!

The `LeftCircle` was already been created and use into the DOM to specific the element, so not really need the `LeftCell` to indicated the `Circle`. 

eg:

```
export const LeftCircle = styled(Circle)``;

 <LeftCell>
  <LeftCircle // ---------> specific use for LeftCircle even it inherit for Circle styled.
     $position="left"
     $variant={variant}
   />
</LeftCell>
```